snapcraft/_store.py

```diff
@@ -111,6 +112,58 @@ def upload(snap_filename):
     tracker.raise_for_code()
 
 
+def _get_text_for_opened_channels(opened_channels):
+    if len(opened_channels) == 1:
+        return 'The {!r} channel is now open.'.format(opened_channels[0])
+    else:
+        text = ''
+        for channel in opened_channels[:-1]:
+            if text:
+                text = '{}, {!r}'.format(text, channel)
+            else:
+                text = '{!r}'.format(channel)
+        text = '{} and {!r}'.format(text, opened_channels[-1])
+        return 'The {} channels are now open.'.format(text)
+
+
+def _get_text_for_channel(channel):
```

Comment:
This should return a tuple of three items, not a list. So instead of [channel, version, revision], use (channel, version, revision)